src/pg_supa--0.1.0.sql

```diff
@@ -0,0 +1,4 @@
+create function get_one()
+returns int as
+$$ select 1;
+$$ language sql immutable strict;
```

Comment:
How about moving the sql functions to a `sql` directory? Usually the `src` folder only contains C code. This can be seen on [timescaledb](https://github.com/timescale/timescaledb) and [pgvector](https://github.com/ankane/pgvector), for example.